### PR TITLE
Small grafana tweaks

### PR DESCRIPTION
**Pull request by garath**
- some doc changes I had yet to commit
- disable snapshots to external server. Snapshots can be useful and I think we'll use them more. There's no reason to use an external server to host them.

### CODE REVIEW

File: src/Monitoring/grafana-init/grafana.ini [hunk 1]
```diff
@@ -69,3 +69,6 @@ provider = azure_blob
 account_name = dotnetgrafana
 ;account_key = <managed by environment>
 container_name = grafana
+
+[snapshots]
+external_enabled = false
```
Comment (ChadNedzlek) [inline src/Monitoring/grafana-init/grafana.ini]: What's this? Should we re-run the setup to get it pushed to the servers? (I think that will upgrade Kusto, which will cause other problems)

Comment (garath) [reply src/Monitoring/grafana-init/grafana.ini]: On a dashboard, click the "Share" button, the select the "Snapshot" tab. This option turns off the "Publish to snapshot.raintree.io". It's not a big deal, but it seems best to not let our data roam around unless necessary. 

Pushing this out would be good just to have a consistent setup but this is not urgent.

**Comment by ChadNedzlek** [comment]
@garath Any reason you haven't merged this yet?

**Comment by garath** [comment]
> @garath Michael Stuckey FTE Any reason you haven't merged this yet?

Whoops... merged now.

**Pull request merged**